.github/ISSUE_TEMPLATE/010-specification-review.md

```diff
@@ -10,49 +10,53 @@ assignees: ''
-
-  - Explainer¹ (minimally containing user needs and example code): [url]
-  - Specification URL: [spec url]
-  - Tests: [wpt folder(s), if available]
-  - User research: [url to public summary/results of research]
-  - Security and Privacy self-review²: [url]
-  - GitHub repo: [url]
-  - Primary contacts (and their relationship to the specification):
-      - [name] ([github username]), [organization/s] (repeat as necessary, we recommend including group chairs and editors in this list)
-  - Organization(s)/project(s) driving the specification: [organization and/or project name]
-  - Key pieces of existing multi-stakeholder (e.g. developers, implementers, civil society) support, review or discussion of this specification
-  - Key pieces of multi-implementer support:
-    - Chromium comments: [url]
+I'm requesting a TAG review of {{feature}}.
+
+{{One paragraph summary of the feature, ideally copy-pasted from your Explainer.}}
+
+  - Explainer¹:
+  - Specification:
+  - WPT Tests:
+  - User research:
+  - Security and Privacy self-review²:
+  - GitHub repo:
+  - Primary contacts:
+      - $name (@-mention), $organization/s, $role in developing specification
+      - {{repeat as necessary, we recommend including group chairs and editors in this list}}
+  - Organization/project driving the specification:
+  - Multi-stakeholder support³:
+    - Chromium comments:
     - Mozilla comments: https://github.com/mozilla/standards-positions/issues/NNN
     - WebKit comments: https://github.com/WebKit/standards-positions/issues/NNN
-    - etc...
-  - External status/issue trackers for this specification (publicly visible, e.g. Chrome Status):
+    - {{...include feedback/review from developers, implementers, civil society, and others}}
+  - Status/issue trackers for implementations⁴:
 
 Further details:
 
   - [ ] I have reviewed the TAG's [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
-  - Relevant time constraints or deadlines: [please provide]
+  - Relevant time constraints or deadlines:
   - The group where the work on this specification is currently being done:
-  - The group where standardization of this work is intended to be done (if current group is a community group or other incubation venue):
+  - The group where standardization of this work is intended to be done:
   - Major unresolved issues with or opposition to this specification:
   - This work is being funded by:
 
 You should also know that...
 
-[please tell us anything you think is relevant to this review]
+{{Please tell us anything else you think is relevant to this review.}}
 
 ------------------------------------------------------------------------------------
 CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING
 
 Please preview the issue and check that the links work before submitting.
 
+Use links to content rather than pasting text into this issue.  Issues are ephemeral and most of the material we are asking for has long term value.
```

Comment:
I feel like the paragraph before this and the paragraph after should be next to each other. Maybe this one belongs first?